index.html

```diff
@@ -36,12 +45,79 @@
+                </div>
+            </div>
+        </section>
+        <section class="printer">
+            <div class="printer-left">
+                <div class="bills-title">Automate bills, invoices and bookkeeping at no cost</div>
+                <div class="bills-description">
+                    <p class="smallest-text">With Koodaa, all hedache-causing and tedious back office tasks are automated and fully taken care of. Bookkeeping happens automatically as you spend. To log expenses, simply take a receipt with your smartphone.</p>
+                </div>
+            </div>
+            <div class="printer-right">
+                <img src="./images/phone_print.png">
+            </div>
+        </section>
+
+        <section class="coins">
+            <div class="coins-left">
+                <img src="./images/phone_cash.png">
+            </div>
+            <div class="coins-right">
+                <div class="coins-title">Instantly send and receive money across border</div>
+                <div class="coins-description">
+                    <p class="smallest-text">By integrating Bash, Kooda enables users to transact across borders instantaneously and without the complexity of international banking. All users accounts are fundet using Dash.</p>
+                </div>
+            </div>
+        </section>
+
+
+        <section class="payment">
+            <div class="payment-left">
+                <div class="payment-title">Cut your payment cycle from months to seconds</div>
+                <div class="payment-description">
+                    <p class="smallest-text">With Koodaa, all hedache-causing and tedious back office tasks are automated and fully taken care of. Bookkeeping happens automatically as you spend.</p>
+                </div>
+            </div>
+            <div class="payment-right">
+                <img src="./images/phone_bill.png">
+            </div>
+        </section>
+
+        <section class="wand">
+            <div class="wand-left">
+                <img src="./images/harry_potter.png">
+            </div>
+            <div class="wand-right">
+                <div class="wand-title">Instantly send and receive money across border</div>
+                <div class="wand-description">
+                    <p class="smallest-text">By integrating Bash, Kooda enables users to transact across borders instantaneously and without the complexity of international banking. All users accounts are fundet using Dash.</p>
+                </div>
+            </div>
+        </section>
 
+<section class="slider">
+    <div>first slide</div>
+    <div>second slide</div>
+    <div>third slide </div>
+</section>
 
         <div></div>
 
 
         <script src="script.js" type="text/javascript"></script>
+        <script type="text/javascript" src="//code.jquery.com/jquery-1.11.0.min.js"></script>
```

Comment:
jak domergujesz ten PR -> https://github.com/micpot/koodaa/pull/1/files. To automatycznie te linki będą poprawione